### ISSUE TITLE
Add Dockerfile

### ISSUE THREAD

**Issue by rkoeninger** [Shen-Language/shen-cl#21]
* Add a Dockerfile to the repo root that copy Linux SBCL build into container.
  * Can be based on small alpine image
  * Can host different image for each release of each port
  * Could also just refer to `shen-cl` build as `shen` as that's the most widely used one
  * Just shen-cl for now?
* Make `shen` organization on DockerHub, name is still available

Publicly hosted docker image would allow one to start working with Shen by running:

```
docker run -it --rm shen/shen:latest
```

**Comment by tizoc** [comment]
Maybe an image based on alpine would be a better option in terms of size.

**Comment by rkoeninger** [comment]
Oh, I assumed centos was one of the smaller ones. [Alpine is much smaller](https://nickjanetakis.com/blog/the-3-biggest-wins-when-using-alpine-as-a-base-docker-image).

**Comment by tizoc** [comment]
Yes, and if you do a [multi-staged build](https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds) the resulting image should be not much bigger than the resulting Shen execultable.

**Comment by tizoc** [comment]
This one looks like a good base, uses Alpine: https://github.com/eshamster/docker-cl-base
